src/main/java/uk/ac/cam/cl/dtg/segue/api/QuestionFacade.java

```diff
@@ -232,6 +261,69 @@ public Response getQuestionsAnswered(@Context final HttpServletRequest request,
     }
   }
 
+  /**
+   * REST end point to provide five questions in random located in the question tile of the student dashboard.
```

Comment:
The backend should not care how the frontend uses the data it returns, so I would remove every mention of the frontend and make this description more generic.